scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/Problem.java

```diff
@@ -18,7 +17,7 @@
    */
   private final MissionModel<?> missionModel;
 
-  private final SimulationFacade simulationFacade;
+  private SimulationFacade simulationFacade;
```

Comment:
I think the removal of `final` here may have been on accident?

Reply:
Yes it has been an accident, I'll put it again.